package.json

```diff
@@ -18,20 +18,21 @@
     "tslint": "tslint"
   },
   "dependencies": {
+    "@angular/cli": "^6.0.3",
```

Comment:
This should be in devDependencies , not in dependencies

Reply:
Good catch, dunno why it can in dependencies instead of devDependencies.